src/main/java/Main.java

```diff
@@ -58,14 +58,26 @@ public static void main(String[] args) {
         int matchScore;
         int scoreIMP;
         Main obj = new Main();
-        obj.getHandValueAndStage();
-        valueOfHand = obj.calculateValueOfHand();
-        System.out.println("Ta reka warta jest: " + valueOfHand);
-        obj.getMatchResults();
-        matchScore = obj.calculateMatchScore();
-        System.out.println("Ilosc zdobytych punktow: " + matchScore);
-        scoreIMP = obj.getResultInInternationalMatchPoints(matchScore, valueOfHand);
-        System.out.println("Ilosc zdobytych punktow w miedzynarodowej notacji sportowej: " + scoreIMP);
+        try {
+            obj.getHandValueAndStage();
+            valueOfHand = obj.calculateValueOfHand();
+            System.out.println("Ta reka warta jest: " + valueOfHand);
+            obj.getMatchResults();
+            matchScore = obj.calculateMatchScore();
+            System.out.println("Ilosc zdobytych punktow: " + matchScore);
+            scoreIMP = obj.getResultInInternationalMatchPoints(matchScore, valueOfHand);
+            System.out.println("Ilosc zdobytych punktow w miedzynarodowej notacji sportowej: " + scoreIMP);
+        } catch (IncorrectHandValueException e) {
```

Comment:
Jeżeli w każdym catchu robisz to samo możesz je wszystkie pogrupować i złapać tylko raz. Dodatkowo zastanów się, czy chcesz pokazywać całego stack trace czy tylko message. Stack trace raczej nie jest potrzebny użytkownikowi docelowemu.